.github/workflows/post-merge-doc-sync.yml

```
@@ -0,0 +1,182 @@
+        with:
+          node-version: '20'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Get merged PR details
+        id: pr
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)